community_supplied/Kernel/Multicast/check-multicast-iif-mismatch-error-cnt-iagent.rule

```diff
@@ -0,0 +1,103 @@
+/*
+ * This rule checks monitors the health of the system on the basis of the iif_mismatch_err_cnt parameter.
```

Comment:
Modify as below 
s/This rule checks monitors the health of the system on the basis of the iif_mismatch_err_cnt parameter./This rule collects iif mismatch error count and notify anomaly if iif mismatch error count exceeds threshold/g